platforms/posix/CMakeLists.txt

```diff
@@ -101,6 +101,14 @@ endif()
 if("${PX4_BOARD}" MATCHES "beaglebone_blue")
 	target_link_libraries(px4 PRIVATE robotics_cape)
 
+elseif("${PX4_BOARD}" MATCHES "modalai_voxl2")
```

Comment:
We should invent some mechanism that pushes this into the board directory.

(note for later)